src/Microsoft.Health.Fhir.Core/Exceptions/RequestEntityTooLargeException.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Health.Fhir.Core.Exceptions
     public class RequestEntityTooLargeException : FhirException
     {
         public RequestEntityTooLargeException()
+            : base(Resources.RequestEntityTooLargeException)
```

Comment:
I think it's fine to make the message not required to be null in the base MicrosoftHealthException. The Exception class does not require it to be non-null anyways and you don't have to repeat these messages everywhere that's not going to be used.

Reply:
That makes sense. I will remove the Debug.Assert from MicrosoftHealthException in shared components code.

Reply:
Updated.